website/docs/r/worklink_fleet.html.markdown

```diff
@@ -0,0 +1,92 @@
+---
+layout: "aws"
+page_title: "AWS: aws_worklink_fleet"
+sidebar_current: "docs-aws-resource-worklink-fleet"
+description: |-
+  Provides a AWS WorkLink Fleet resource.
+---
+
+# aws_worklink_fleet
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "aws_worklink_fleet" "example" {
+  name  = "terraform-example"
+}
+```
+
+Network Configuration Usage:
+
+```hcl
+resource "aws_worklink_fleet" "example" {
+  name = "terraform-example"
+
+  network {
+		vpc_id = "${aws_vpc.test.id}"
+		subnet_ids = ["${aws_subnet.test.*.id}"]
+		security_group_ids = ["${aws_security_group.test.id}"]
+  }
+}
+```
+
+Identity Provider Configuration Usage:
+
+```hcl
+resource "aws_worklink_fleet" "test" {
+	name = "tf-worklink-fleet-%s"
+
+	identity_provider {
+		type = "SAML"
+		saml_metadata = "${file("saml-metadata.xml")}"
+	}
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) A region-unique name for the AMI.
+* `display_name` - (Optional) The name of the fleet.
```

Comment:
Nit: The optional arguments should be alphabetized along with the configuration block sections below.

Reply:
Fixed